api-scripts/one-off-scripts/retrieving-datahub-datasets.py

```diff
@@ -0,0 +1,87 @@
+"""A script that retrieves and lists Public Data Hub Datasets.
+The output will contain a list of dataset titles, their creation date,
+and their placenames. This script was created to facilitate the tracking of
+dataset addition dates, as well as spatial spread of data, for Data
+Hub Analytics.
+
+To Run From data.naturalcapitalproject.stanford.edu repo:
+    python api-scripts/one-off-scripts/retrieving-datahub-datasets.py
+
+Dependencies:
+    $ mamaba install requests json
```

Comment:
Won't this fail?  `json` is part of the standard library and there isn't a conda-forge package called `json` probably for this reason.
```suggestion
    $ mamaba install requests
```